test/NuGet.Core.FuncTests/NuGet.Commands.FuncTest/RestoreCommand_PrunePackageReference.cs

```diff
@@ -1283,19 +1283,24 @@ public void AnalyzePruningResults_WithVariousFrameworks_WarnsForNET10OrNewerOnly
 
             var projectSpec = ProjectTestHelpers.GetPackageSpecWithProjectNameAndSpec("Project1", Path.GetTempPath(), rootProject);
             var testLogger = new TestLogger();
+            var testEvent = new TelemetryEvent("dummyEvent");
 
-            RestoreCommand.AnalyzePruningResults(projectSpec, testLogger);
+            RestoreCommand.AnalyzePruningResults(projectSpec, testEvent, testLogger);
 
             if (shouldWarn)
             {
                 testLogger.WarningMessages.Should().HaveCount(1);
                 var restoreLogMessage = (RestoreLogMessage)testLogger.LogMessages.Single();
                 restoreLogMessage.Code.Should().Be(NuGetLogCode.NU1510);
                 restoreLogMessage.LibraryId.Should().Be("A");
+                testEvent["Pruning.RemovablePackages.Count"].Should().Be(1);
```

Comment:
nit: if these aren't relevant to the Warning being shown, they could be moved outside this conditional.

Reply:
I honestly thought about it and I forgot to do it. I'll do it in a new push.

Reply:
fixed